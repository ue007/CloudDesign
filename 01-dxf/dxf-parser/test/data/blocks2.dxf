  0
SECTION
  2
BLOCKS
  0
BLOCK
5
1
8
0
2
$MODEL_SPACE
70
0
10
0.0
20
0.0
30
0.0
3
$MODEL_SPACE
1

0
ENDBLK
5
21
8
0
0
BLOCK
5
2
67
1
8
0
2
$PAPER_SPACE
70
0
10
0.0
20
0.0
30
0.0
3
$PAPER_SPACE
1

0
ENDBLK
5
1D
67
1
8
0
0
ENDSEC
0
EOF