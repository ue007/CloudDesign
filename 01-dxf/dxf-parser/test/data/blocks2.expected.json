{
  "blocks": {
    "$MODEL_SPACE": {
      "handle": "1",
      "layer": "0",
      "name": "$MODEL_SPACE",
      "position": {
        "x": 0,
        "y": 0,
        "z": 0
      },
      "name2": "$MODEL_SPACE",
      "xrefPath": ""
    },
    "$PAPER_SPACE": {
      "handle": "2",
      "paperSpace": true,
      "layer": "0",
      "name": "$PAPER_SPACE",
      "position": {
        "x": 0,
        "y": 0,
        "z": 0
      },
      "name2": "$PAPER_SPACE",
      "xrefPath": ""
    }
  }
}